string robot_id
string state
---
string confirm_update
